src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs

```diff
@@ -65,6 +77,11 @@ bool Execute (DirectoryAssemblyResolver resolver)
 
 			var topAssemblyReferences = new List<AssemblyDefinition> ();
 
+			LockFile lockFile = null;
+			if (!string.IsNullOrEmpty (ProjectAssetFile) && File.Exists (ProjectAssetFile)) {
+				lockFile = LockFileUtilities.GetLockFile (ProjectAssetFile, NullLogger.Instance);
```

Comment:
Do we really want `NullLogger.Instance` here? That implies we won't get any log messages from NuGet, or so I assume.